src/components/LoginScreen.js

```diff
@@ -1,11 +1,25 @@
 import React, { Component } from 'react';
 import { View, TextInput, Button, Text } from 'react-native';
 import { openProductList } from '../actions/NavigationActions';
+import { changeEmail, changePassword, resetLoginData, loginAction } from '../actions/LoginActions';
 import { connect } from 'react-redux';
 import { styles } from './styles';
 
 export class LoginScreen extends Component {
+    
+    goToProductList = () => {
+        this.props.loginAction();
+    }
+
+    componentDidUpdate() {
+        if(this.props.login.userLogedIn) {
+            const { navigate } = this.props.navigation;
+            navigate("ProductsList");
```

Comment:
It's better to store the screen names as constants, and then import and use them whenever you need.